File: 윤호선/34주차/숫자게임.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> A, vector<int> B) {
    int answer = 0;
    
    sort(A.begin(),A.end());
    sort(B.begin(),B.end());
    
    int idx=B.size()-1;
    for(int i=A.size()-1;i>=0;i--){
        if(A[i]<B[idx]){
            answer++;
            idx--;
        }
    }
    return answer;
}

File: 노영주/34주차/숫자게임.js
const solution = (A, B) => {
  let answer = 0;

  A.sort((a, b) => b - a);
  B.sort((a, b) => a - b);

  let idx = B.length - 1;

  A.forEach((member) => {
    if (member < B[idx]) {
      answer++;
      idx--;
      B.pop();
    }
  });

  return answer;
};
